.github/CODEOWNERS

```diff
@@ -7,4 +7,4 @@
 # global owners are only requested if there isn't a more specific
 # codeowner specified below. For this reason, the global codeowners
 # are often repeated in package-level definitions.
-*       @tzdybal @liamsi @adlerjohn
+*       @tzdybal
```

Comment:
Should John and I be added to the docs sub-package though? Or at least to: https://github.com/celestiaorg/optimint/tree/main/docs/lazy-adr
?